### ISSUE TITLE
when training for cfg, Why only utilize half of the input？

### ISSUE THREAD

**Issue by jinge170** [facebookresearch/DiT#53]
half = x[: len(x) // 2]
combined = torch.cat([half, half], dim=0)
